wezterm-gui/src/overlay/selector.rs

```diff
@@ -178,25 +189,46 @@ impl SelectorState {
     }
 
     fn run_loop(&mut self, term: &mut TermWizTerminal) -> anyhow::Result<()> {
+        let max_items = self.max_items;
+        let alphabet = self.args.alphabet.to_lowercase();
+        let alphabet_has_j = alphabet.contains("j");
```

Comment:
This seems like a smart approach. Should `/` also have similar treatment? Are there other unmodified `Char` patterns below that might also need similar treatment?

FWIW, I think it would be ok to inline this check where the event is being matched; I'm less concerned about the runtime cost of `alphabet.contains` than I am about the readability of this function, and I don't think that having a separate local variable that is many lines apart from the logic helps with readability.

Reply:
I have inlined it as `self.args.alphabet.contains("j")` now.

Reply:
To do something similar for `/`, we would need to add another way to search, I think. I am not sure it makes sense to add that.